scripts/validate_documentation.py

```diff
@@ -0,0 +1,227 @@
+import argparse
+from fnmatch import fnmatch
```

Comment:
haven't used `fnmatch`, but I prefer to import modules with submodules having the same name (like `datetime`) as `import fnmatch`. Then, in the code, when you see `fnmatch.fnmatch` there is no ambiguity whether it's the module or the submodule.